### PR TITLE
Issue #6503: Add icon to language switcher in admin bar

### LINKED ISSUE #6503: Follow-up: Add icon to language switcher in admin bar
## Description of the bug

Another little follow-up, this time relevant for sites with the **locale** module enabled (multilingual sites).

The language switcher needs an icon, too. Currently it's the default, which doesn't fit well.

This one:
language-icon-missing

Easy to fix, now that we have an API for icons. And loads of icons. :smile: 

## Additional information

- Backdrop CMS version: 1.28.x

### PR DESCRIPTION
**Pull request by indigoxela**
Fixes https://github.com/backdrop/backdrop-issues/issues/6503

### CODE REVIEW

**Review by klonos — APPROVED** [review]
LGTM 👍🏼

File: core/modules/admin_bar/admin_bar.inc [hunk 1]
```diff
@@ -752,6 +752,10 @@ function admin_bar_links_locale() {
   $links['locale'] = array(
     '#title' => isset($language->native) ? $language->native : $language->name,
     '#attributes' => array('class' => array('admin-bar-locale-links')),
+    '#options' => array(
+      'html' => TRUE,
```
Comment (quicksketch) [inline core/modules/admin_bar/admin_bar.inc]: I don't think `'html'` is needed here, and might actually cause an XSS issue if the language name (which is configurable) is printed without sanitization.

Comment (indigoxela) [reply core/modules/admin_bar/admin_bar.inc]: Oops, right, that unnecessary and possibly dangerous. Removed. :+1:

**Comment by quicksketch** [comment]
Thanks @indigoxela and @klonos! Merged into 1.x for 1.28.0.

**Pull request merged**